converter/internal/staticconvert/internal/build/builder.go

```diff
@@ -193,6 +195,85 @@ func (b *IntegrationsConfigBuilder) appendExporter(commonConfig *int_config.Comm
 	b.globalCtx.InitializeRemoteWriteExports()
 }
 
+func (b *IntegrationsConfigBuilder) appendV2Integrations() {
+	for _, integration := range b.cfg.Integrations.ConfigV2.Configs {
+		var exports discovery.Exports
+		var commonConfig common_v2.MetricsConfig
+
+		switch itg := integration.(type) {
+		case *agent_exporter_v2.Config:
+			exports = b.appendAgentExporter(itg)
```

Comment:
This feels off and maybe a comment/naming would fix it. In a lot of places v2 is used but this looks wrong especially with the appendExporterV2 a few lines below. Looking into the func it pivots but feels like friction here.